src/test/kotlin/no/nav/tilleggsstonader/sak/brev/mellomlager/MellomlagringBrevServiceTest.kt

```diff
@@ -0,0 +1,78 @@
+package no.nav.tilleggsstonader.sak.brev.mellomlager
+
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkObject
+import io.mockk.unmockkObject
+import no.nav.tilleggsstonader.sak.infrastruktur.sikkerhet.SikkerhetContext
+import org.assertj.core.api.AssertionsForClassTypes.assertThat
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.springframework.data.repository.findByIdOrNull
+import java.util.UUID
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Hvorfor trengs denne her?

Reply:
Den hang med fra EF, men vil tror ikke `@BeforeAll` og `@AfterAll` funker uten. Så var vel sikkert for å kun måtte mocke `SikkerhetContext` en gang, istedenfor å gjøre det for hver test 🤷‍♂️